friendsofsymfony/user-bundle/2.1/config/packages/fos_user.yaml

```diff
@@ -0,0 +1,9 @@
+fos_user:
+    db_driver: orm
+    service:
```

Comment:
Can you move `service` block between `user_class` and `from_email` for more readability please?